public/js/components/Editor.js

```diff
@@ -77,6 +81,13 @@ const Editor = React.createClass({
     }
   },
 
+  resizeBreakpointGutter() {
```

Comment:
The technique here anchors the breakpoint to the beginning of the line, or the "right side" of the line number gutter. That way if there are 10,000,000 lines and the gutter expands really large, the breakpoint will be anchored to the right side and will travel with it. Unfortunately I need to do a little manual layout for that.